libs/core/src/lib/fixed-card-layout/fixed-card-layout.component.ts

```diff
@@ -23,15 +22,15 @@ import {
     ViewEncapsulation
 } from '@angular/core';
 import { FocusKeyManager } from '@angular/cdk/a11y';
-import { CdkDrag, CdkDragDrop, CdkDragEnter, CdkDropList } from '@angular/cdk/drag-drop';
-import { Subject } from 'rxjs';
-import { debounceTime, delay, takeUntil } from 'rxjs/operators';
+import { CdkDrag, CdkDragDrop, CdkDragEnter } from '@angular/cdk/drag-drop';
+import { Subject, Subscription } from 'rxjs';
+import { debounceTime, delay, distinct, takeUntil } from 'rxjs/operators';
 
 import { resizeObservable, RtlService } from '@fundamental-ngx/core/utils';
 import { FixedCardLayoutItemComponent } from './fixed-card-layout-item/fixed-card-layout-item.component';
 import { coerceNumberProperty, NumberInput } from '@angular/cdk/coercion';
 
-const PX_IN_REM = 16;
+const REM_IN_PX = 16;
```

Comment:
I we want to be more precise about it, we should get document's font-size computed value